auth/login/auth-login.component.ts

```diff
@@ -127,6 +131,20 @@ export class AuthLoginComponent {
         }
     }
 
+    openPrivacyLink() {
+        switch(this.local_i18n.language){
```

Comment:
with the switch, you have to adjust this block for each additional language.
And each project using this feature would have to apply to the convention to only have a lang-parameter for "en" and not for "de", for example. Will be problematic for a non-german company.

I would rather like sth. as
`window.open(environment.zeta.privacyLink + '&lang=' + this.i18n.language, '_blank').focus();`.

Another, more flexible option is to place a `privacyLink: function(lang: string): string` into the environment instead of only the string itself